server/src/test/java/org/opensearch/cluster/node/DiscoveryNodesTests.java

```diff
@@ -108,12 +108,12 @@ public void testAll() {
         assertThat(discoveryNodes.resolveNodes(new String[0]), arrayContainingInAnyOrder(allNodes));
         assertThat(discoveryNodes.resolveNodes("_all"), arrayContainingInAnyOrder(allNodes));
 
-        final String[] nonMasterNodes = StreamSupport.stream(discoveryNodes.getNodes().values().spliterator(), false)
+        final String[] nonClusterManagerNodes = StreamSupport.stream(discoveryNodes.getNodes().values().spliterator(), false)
             .map(n -> n.value)
             .filter(n -> n.isMasterNode() == false)
```

Comment:
`isMasterNode` -> `isClusterManagerNode`

Reply:
`isMasterNode()` is a public method, so I plan to change it in the next phase when renaming all the public Java API, and targeted for 3.0 version.